code/src/App.js

```diff
@@ -1,9 +1,138 @@
-import React from 'react'
+import React, { useState } from "react"
```

Comment:
The questions are really nice and I liked the idea behind the app. There are so many ways to import/use components and I think that the way you choose to go about it is just fine (importing header and summary in the App.js folder and creating the forms/inputs in App.js. A step ahead would be creating separate components for each question and importing them inside App.js one by one. Both ways work!🧠